### ISSUE TITLE
Less verbose `s/form` on Specs

### ISSUE THREAD

**Issue by ikitommi** [metosin/spec-tools#159]
There is no need to publish the `spec-tools.parse` namespaced keys as they are re-created when spec is created (from a form). 

```clj
(require '[spec-tools.data-spec :as ds])
(require '[clojure.spec.alpha :as s])

(def weather-spec
  (ds/spec
    {:spec
     {:weather {:weatherbitio {:key string? :default {:zip string?}}}
      :command {:prefix string?}}
     :name ::weather}))

(s/form weather-spec)
; (spec-tools.core/spec
;  {:spec (clojure.spec.alpha/keys :req-un [:user$weather/weather :user$weather/command]),
;   :type :map,
;   :spec-tools.parse/key->spec {:weather :user$weather/weather, :command :user$weather/command},
;   :spec-tools.parse/keys #{:command :weather},
;   :spec-tools.parse/keys-req #{:command :weather},
;   :name :user/weather})
```

could be: 

```clj
(s/form weather-spec)
; (spec-tools.core/spec
;  {:spec (clojure.spec.alpha/keys :req-un [:user$weather/weather :user$weather/command]),
;   :type :map,
;   :name :user/weather})
```

**Comment by Biserkov** [comment]
This issue is fixed by [0.9.0](https://github.com/metosin/spec-tools/blob/fc0b3b6e86cf2619eac725afc30f0dfa082bb613/CHANGELOG.md#090-2622019)

**Comment by ikitommi** [comment]
True that!
